serving/src/main/resources/application.conf

```diff
@@ -73,8 +73,23 @@ defaults {
     ssl-verify = false
     ssl-verify = ${?SSL_VERIFY_CONFIG}
     # SSL Cert Configs for Git
-  }
 
+    #S3 settings
+    # these are application specific settings that dont belong to the S3Client api;
+    # the S3Client api's settings are overriden programmatically
+    s3 {
+      # to change the download-timeout limit, please refer to the setting 'side-effects-delivery-timeout'
+      download-timeout = ${defaults.application.proxy.side-effects-delivery-timeout}
```

Comment:
Please add an environment variable override option while keeping ${defaults.applicatin...} as the first, default options.
Also, please add to the comment here that side-effects-delivery-timeout is an upper bound for the download-timeout